rs/sns/governance/src/types.rs

```diff
@@ -2752,17 +2752,25 @@ pub mod test_helpers {
 
 impl From<upgrade_journal_entry::UpgradeStepsRefreshed> for upgrade_journal_entry::Event {
     fn from(event: upgrade_journal_entry::UpgradeStepsRefreshed) -> Self {
-        upgrade_journal_entry::Event::UpgradeStepsRefreshed(event)
+        Self::UpgradeStepsRefreshed(event)
     }
 }
+
+impl From<upgrade_journal_entry::UpgradeStepsReset> for upgrade_journal_entry::Event {
+    fn from(event: upgrade_journal_entry::UpgradeStepsReset) -> Self {
+        Self::UpgradeStepsReset(event)
+    }
+}
```

Comment:
This will need to be moved into src/upgrade_journal.rs, also please implement helpers in that file for `UpgradeStepsReset` (should pretty much be a copy and paste job from `UpgradeStepsRefreshed`).